build_tools/benchmarks/comparisons/mobilebert_fp32_commands.py

```diff
@@ -0,0 +1,156 @@
+    #yapf: enable
+
+
+class MobilebertFP32CommandFactory(BenchmarkCommandFactory):
+  """ Generates `BenchmarkCommand` objects specific to running MobileBert."""
+
+  def __init__(self, base_dir: str):
+    self._model_name = "mobilebert_float_384_gpu"
+    self._base_dir = base_dir
+    self._iree_benchmark_binary_path = os.path.join(base_dir,
+                                                    "iree-benchmark-module")
+    self._tflite_benchmark_binary_path = os.path.join(base_dir,
+                                                      "benchmark_model")
+    self._tflite_model_path = os.path.join(self._base_dir, "models", "tflite",
+                                           self._model_name + ".tflite")
+    self._tflite_test_data_dir = os.path.join(self._base_dir, "test_data",
+                                              "squad")
+
+  def generate_benchmark_commands(self, device: str,
+                                  driver: str) -> list[BenchmarkCommand]:
+    if device == "desktop" and driver == "cpu":
+      return self._generate_cpu(device)
+    elif device == "desktop" and driver == "gpu":
+      return self._generate_gpu("cuda")
+    elif device == "mobile" and driver == "cpu":
+      return self._generate_cpu(device)
+    elif device == "mobile" and driver == "gpu":
+      return self._generate_gpu("vulkan")
+    else:
+      print("Warning! Not a valid configuration.")
+      return []
+
+  def _generate_cpu(self, device: str):
+    # Generate TFLite benchmarks.
+    tflite_mobilebert = TfliteMobilebertFP32(self._tflite_benchmark_binary_path,
+                                             self._model_name,
+                                             self._tflite_model_path,
+                                             self._tflite_test_data_dir,
+                                             driver="cpu")
+
+    # Generate IREE benchmarks.
+    driver = "dylib"
+    iree_model_path = os.path.join(self._base_dir, "models", "iree", driver,
+                                   self._model_name + ".vmfb")
+    iree_mobilebert = IreeMobilebertFP32(self._iree_benchmark_binary_path,
+                                         self._model_name,
+                                         iree_model_path,
+                                         driver=driver)
+    commands = [tflite_mobilebert, iree_mobilebert]
+
+    # Test mmt4d only on mobile.
```

Comment:
I think it's alright to test on desktop. We just want to test on desktop with the correct parameters (which can include just the fallback parameters)